packages/app/src/app/pages/Profile/Showcase/SandboxInfo/index.tsx

```diff
@@ -1,19 +1,30 @@
-import React from 'react';
+import Row from '@codesandbox/common/lib/components/flex/Row';
+import { Sandbox } from '@codesandbox/common/lib/types';
 import { getSandboxName } from '@codesandbox/common/lib/utils/get-sandbox-name';
 import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
-import Row from '@codesandbox/common/lib/components/flex/Row';
+import React, { FunctionComponent } from 'react';
+
 import { Stat } from 'app/components/Stat';
-import SvgButton from './play-button.svg';
+import { useOvermind } from 'app/overmind';
+
 import {
   Container,
-  Title,
-  Like,
   Description,
-  Stats,
+  Like,
   PlayButtonContainer,
+  Stats,
+  Title,
 } from './elements';
+import SvgButton from './play-button.svg';
+
+type Props = {
+  sandbox: Sandbox;
+};
+export const SandboxInfo: FunctionComponent<Props> = ({ sandbox }) => {
+  const {
+    state: { isLoggedIn },
+  } = useOvermind();
```

Comment:
Hmm, I do like the fact that this component is "standalone" without the store, so I'd prefer to keep `useOvermind` out if it's not really needed.

Reply:
This component is only used once _and_ we've done a lot of refactors where we used `useOvermind` as near as possible to the needing code.